app/components/Onboarding/Syncing.js

```diff
@@ -9,10 +9,9 @@ import styles from './Syncing.scss'
 
 class Syncing extends Component {
   componentWillMount() {
-    const { fetchBlockHeight, newAddress } = this.props
+    const { fetchBlockHeight } = this.props
 
     fetchBlockHeight()
-    newAddress('np2wkh')
```

Comment:
@mrfelton removing this is great. on current master this is called (`componentWillMount`) _before_ gRPC is connected so the call to get a new address hangs and we never get one. 

Instead of calling for a new address on the `componentWillMount` callback I suggest we `dispatch(fetchInfo())` (which also gets a new address according to this PR) in the `grpcConnected` callback here: https://github.com/LN-Zap/zap-desktop/blob/master/app/reducers/lnd.js#L46

Reply:
I patched the above suggestion locally and it works great ^